tests/src/log4net.Tests.csproj

```diff
@@ -19,245 +18,25 @@
  under the License.
 
 -->
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectType>Local</ProjectType>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{B0530F10-0238-49A9-93B0-8EF412E90BCF}</ProjectGuid>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ApplicationIcon>
-    </ApplicationIcon>
-    <AssemblyKeyContainerName>
-    </AssemblyKeyContainerName>
+    <!--
+         Currently multitargeting tests doesn't work, separate projects are required.
+         (tracked at https://github.com/Microsoft/vstest/issues/298 and https://github.com/Microsoft/vstest/issues/624)
+    -->
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
I agree about nant. However, it is nice that it's stable for the .NET Framework and Mono builds, and works well enough in your CI. Building entire releases from one project is somewhat complex here because log4net has so many targets. In particular, I don't know enough about Mono <--> .NET Framework interop to know whether MSBuild can reasonably build for both in a Linux environment.